index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO MICROVERSE</title>
+    <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="wrap-section">
+        <header>
+            <nav>
+                <a href="#" class="my-logo">Aneal Laryea</a>
+                <i class="fa-solid fa-bars"></i>
+            </nav> 
+        </header>
+    
+        <section class="headline">
+            <div class="primary-text">
+                <h3>Hey There.
+                    I’m Aneal</h3>
+            </div>
+            <div class="supporting-text">
+                <p>I’am a Software Developer</p>
+            </div>
+            <div class="description">
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+            </div>
```

Comment:
[OPTIONAL]
Please stick to the Figma design details : color, font, marge

Your design
<img width="343" alt="image" src="https://user-images.githubusercontent.com/7243132/170589468-4eecdeb2-6d33-4042-940b-575e7d6f3cf7.png">

Figma
<img width="343" alt="image" src="https://user-images.githubusercontent.com/7243132/170589506-d5084c48-d5c1-45d8-8dde-b2981a8beace.png">

Reply:
@jaflih what can I do to stick with the design in this section?